app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] Similarly, Comment's `id` should reference the foreign key `author_id` (declared in the _User_ model) of the _User_, and hence it's redundant to declare `author_id` as foreign key in _Comment_ model. Simply do something like:
```ruby
# app/models/comment.rb
belongs_to :author, class_name: 'User'